Add semantic convention of influxDB

steverao opened the pull request:
## Changes

Please provide a brief description of the changes here.
Add semantic convention of influxDB. Relative to  https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10850

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [x] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

AlexanderWert commented on model/trace/database.yaml:
```diff
@@ -251,3 +251,24 @@ groups:
+    brief: >
+      Attributes for InfluxDB
+    attributes:
+      - ref: db.name
```
Note: `db.name` is being changed in another PR: https://github.com/open-telemetry/semantic-conventions/pull/911

trisch-me commented on .chloggen/influxdb.yaml:
```diff
@@ -0,0 +1,22 @@
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
```
I think new component is used if you create brand new type in registry and in this case  it should be enhancement

@joaopgrassi could you confirm if it's correct?

steverao replied:
> I think new component is used if you create brand new type in registry and in this case it should be enhancement
> 
> @joaopgrassi could you confirm if it's correct?

I finished the changelog according to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#adding-a-changelog-entry
It doesn't look like it contains the distinction between `new_component` and `enhancement`. At first, I though this is a new framework so choose `new_component`. If we discuss the difference between them this time, I think it had better to add some explanation about them in document later：）

steverao replied:
> new_component is a new namespace, in this case this is extension of existing db.

Got it, I modified it now.

lmolkova commented:
@steverao  do you envision adding any specifics about InfluxDB? Like custom attributes, metrics, some non-obvious mapping for common attributes?

If the goal is to add a constant for `db.system`, then it can be done without adding `influxdb.md` or the yaml section for it.
Curious what you think and what's your motivation to add things beyond system constant.
Thanks!

steverao commented:
> @steverao do you envision adding any specifics about InfluxDB? Like custom attributes, metrics, some non-obvious mapping for common attributes?

At present, besides `db.system`, I also added some information about new database framework of InfluxDB, such as attributes and examples in https://github.com/open-telemetry/semantic-conventions/pull/949/files#diff-b0b92e2b139407d14dd05fec354af626034d748c5ced214a074e1e2389cffc2c

steverao commented:
> it does not look like influxDB needs any customization on top of generic attributes and descriptions we have for common DB attributes/spans. Am I correct?

Yes, for the plugin, it don't contain special customization.

> There is no requirement to add a semconv file for each DB.

At first, I saw other database plugins provide relevant YAML document, so I added it for InfluxDB. For this situation, don' you recommend to add its unique YAML document? In the document, I added some explanations for its attributes. I think it  will be better if we conserve it. If you think it's not necessary. I can delete it later.

steverao commented:
> It seems that InfluxDB usage of common attributes is very well aligned with common definitions (database name is

Thank you for your explanation! I deleted other contents except `db.system` constant in InfluxDB YAML document now. Please help me to review whether there are any other problems again?

steverao commented:
It seems there is unstable URL in other document which caused the CI running failed!
![image](https://github.com/open-telemetry/semantic-conventions/assets/25423978/8042d91d-2589-41c7-912a-fea6a7d3f65b)

lmolkova commented on docs/database/influxdb.md:
There is no need to add the file since it does not provide any additional information

steverao replied:
> There is no need to add the file since it does not provide any additional information

done
